tests/api/test_addon_uploads.py

```diff
@@ -95,3 +95,150 @@ def test_upload_unlisted_extension(base_url, selenium, session_auth):
     # get the edit url for the add-on to verify that it was created and visible in devhub
     r = requests.get(resp['edit_url'], cookies={"sessionid": session_auth}, timeout=10)
     assert r.status_code == 200
+
+
+@pytest.mark.serial
+@pytest.mark.create_session('api_user')
+def test_submit_extension_with_invalid_uuid_format(base_url, session_auth):
+    """The UUID format doesn't match the expected format for the uuid value"""
+    uuid = [
+        'some-invalid-uuid',
+        {'upload': 'd4ce752a971b4a5aafcd175122726431'},
+        None,
+        '',
+    ]
```

Comment:
This could be a fixture

Reply:
I don't quite understand why I should turn this into a fixture. Could you elaborate a bit?

Reply:
Aren't you using this in multiple tests?

Reply:
Not with the exact same values. The list items will vary depending on the field I want to validate in that test.